### DISCUSSION TITLE
using my squeeze box radio on my mac mini

### DISCUSSION THREAD

**Discussion by gutniklm** [LMS-Community/slimserver#1200] [General]
everything I want to use my squeeze box radio I have to open my logitech in my mac mini in the application file and start the server.  Is there a way to automatically have it start up.

**Comment by michaelherger** [comment]
Do you want to automatically start the application, or start the Mac whenever you want to use your Radio?

If you don't want to use your Mac in order to use the Radio, I'd recommend you look into installing LMS on something else, like eg. a Raspberry Pi. See eg. https://lyrion.org/getting-started/beginners-guide-lms-on-raspberry-pi/. That uses very little power and can be left running 24/7.

If you want more help, I'd strongly recommend you join the community at https://forums.slimdevices.com.
